### PR TITLE
fix: Time zone data not set in `ParseInstallation`

### PR DESCRIPTION
**Pull request by mbfakourii**
### New Pull Request Checklist
- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).

### Issue Description
fix ParseInstallation timeZone missing

Closes: https://github.com/parse-community/Parse-SDK-Flutter/issues/96

### Approach
Added a new way to get the time zone name

### TODOs before merging
- [x] Add tests
- [ ] Add changes to documentation (guides, repository pages, in-code descriptions)
- [x] A changelog entry

### CODE REVIEW

File: packages/dart/lib/parse_server_sdk.dart [hunk 1]
```diff
@@ -12,6 +12,8 @@ import 'package:path/path.dart' as path;
 import 'package:sembast/sembast.dart';
 import 'package:sembast/sembast_io.dart';
 import 'package:sembast_web/sembast_web.dart';
+import 'package:timezone/data/latest.dart' as tz;
```
Comment (mtrezza) [inline packages/dart/lib/parse_server_sdk.dart:15]: Is it correct that these imports use the same reference `tz`?

**Review by mtrezza — COMMENTED** [review]
Could you add a test?

File: packages/dart/lib/parse_server_sdk.dart (hunk 1 above)
Comment (mbfakourii) [reply packages/dart/lib/parse_server_sdk.dart:15]: > Is it correct that these imports use the same reference `tz`?

Yes, this implementation is based on the [timezone](https://pub.dev/packages/timezone) package

**Comment by mbfakourii** [comment]
> Could you add a test?

test added

**Review by mtrezza — APPROVED** [review]
Looks good!

**Comment by mbfakourii** [comment]
> Looks good! If you would like to add the changelog entry, then we can publish a release.

added

File: packages/dart/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## [3.2.1](https://github.com/parse-community/Parse-SDK-Flutter/compare/dart-3.1.9...dart-3.2.1) (2023-01-14)
```
Comment (mtrezza) [inline packages/dart/CHANGELOG.md:1]: Why is this jumping from `3.1.9` to `3.2.1`? If this is a fix it should to `3.1.10`

Comment (mbfakourii) [reply packages/dart/CHANGELOG.md]: I thought it should start from 1 after 9, that's why I changed it.

I changed it to version 3.1.10

Comment (mtrezza) [reply packages/dart/CHANGELOG.md]: Got it. You may want to take a look at [semver](https://semver.org) which we are following in every Parse Platform repository.

File: packages/dart/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+## [3.1.10](https://github.com/parse-community/Parse-SDK-Flutter/compare/dart-3.1.9...dart-3.1.10) (2023-01-16)
+
+### Bug Fixes
+
+* Time zone data not set in `ParseInstallation` ([#811](https://github.com/parse-community/Parse-SDK-Flutter/pull/811))
```
Comment (mtrezza) [inline packages/dart/CHANGELOG.md:5]:
```suggestion
* Time zone data not set in `ParseInstallation` ([#96](https://github.com/parse-community/Parse-SDK-Flutter/issues/96))
```

**Review by mtrezza — APPROVED** [review]
Looks good!

**Pull request merged**